config/initializers/dfe_analytics.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+DfE::Analytics.configure do |config|
+  # Whether to log events instead of sending them to BigQuery.
+  #
+  # config.log_only = true
+
+  # Whether to use ActiveJob or dispatch events immediately.
+  #
+  # config.async = true
+
+  # Which ActiveJob queue to put events on
+  #
+  config.queue = :dfe_analytics
+
+  # The name of the BigQuery table we’re writing to.
+  #
+  # config.bigquery_table_name = ENV['BIGQUERY_TABLE_NAME']
```

Comment:
do those not need to be set here, or are they the default anyway 🤔 ?

Reply:
They default https://github.com/DFE-Digital/dfe-analytics/blob/6d42dc06a849c659b6b44280c799420273f78223/config/locales/en.yml#L18